data/i18n/hi/hi.toml

```diff
@@ -84,111 +184,88 @@ other = "पिछले न्यूज़लेटर देखें"
+[post_create_issue]
+other = "इशू बनाएं"
 
-# layouts > blog > list
+[prerequisites_heading]
+other = "शुरू करने से पहले"
 
-[layouts_case_studies_list_tell]
-other = "अपनी कहानी बतायें"
+[previous_patches]
+other = "पिछले पैच:"
 
-# layouts > docs > glossary
+[release_date_after]
+other = ")"
 
-[layouts_docs_glossary_description]
-other = "इस शब्दावली का उद्देश्य कुबेरनेट्स शब्दावली की एक व्यापक, मानकीकृत सूची है। इसमें कुबेरनेट्स के लिए विशिष्ट तकनीकी शब्द और उपयोगी संदर्भ प्रदान करने वाले सामान्य शब्द शामिल हैं।"
+[release_date_before]
+other = "(रिलीज़ड: "
 
-[layouts_docs_glossary_filter]
-other = "शब्दों को उनके टैग के अनुसार फ़िल्टर करें"
+# See https://gohugo.io/functions/format/#gos-layout-string
+# For a suitable format
+[release_date_format]
+other = "2006-01-02"
 
-[layouts_docs_glossary_select_all]
-other = "सभी का चयन करे"
+[seealso_heading]
+other = "यह भी देखें"
 
-[layouts_docs_glossary_deselect_all]
-other = "सभी को अचयनित करें"
+[subscribe_button]
+other = "सब्स्क्राइब करें"
 
-[layouts_docs_glossary_aka]
-other = "के रूप में भी जाना जाता है"
+[synopsis_heading]
+other = "संक्षेप में"
 
-[layouts_docs_glossary_click_details_before]
-other = "पर क्लिक करें"
+[thirdparty_message]
+other = """यह खंड अन्य पक्ष परियोजनाओं से जुड़ा है जो कुबेरनेट्स द्वारा आवश्यक कार्यक्षमता प्रदान करते हैं। कुबेरनेट्स परियोजना के लेखक इन परियोजनाओं के लिए जिम्मेदार नहीं हैं। यह पृष्ठ <a href="https://github.com/cncf/foundation/blob/master/website-guidelines.md" target="_blank">CNCF वेबसाइट दिशानिर्देश</a> का अनुसरण करते हुए परियोजनाओं को वर्णानुक्रम में सूचीबद्ध करता है। इस सूची में कोई नई परियोजना जोड़ने से पहले यह <a href="/docs/contribute/style/content-guide/#third-party-content">विषय सूची मार्गदर्शन</a> पृष्ट पढ़के ही परिवर्तन करें।"""
 
-[layouts_docs_glossary_click_details_after]
-other = "किसी विशेष शब्द के लिए लंबी व्याख्या प्राप्त करने के लिए नीचे दिए गए संकेतक।"
-
-# layouts > docs > search
+[thirdparty_message_edit_disclaimer]
+other="""अन्य पक्ष के विषयों की सलहा"""
 
-[layouts_docs_search_fetching]
-other = "परिणाम लाए जा रहे हैं.."
+[thirdparty_message_single_item]
+other = """&#128711; यह वस्तु कोई अन्य पक्ष की परियोजना या उत्पाद से जुड़ा है जो कुबेरनेट्स का हिस्सा नहीं हैं। <a class="alert-more-info" href="#third-party-content-disclaimer">अधिक जानकारी के लिए पढ़ें</a>"""
 
-# layouts > partial > feedback
+[thirdparty_message_disclaimer]
+other = """<p>इस पृष्ठ की वस्तुओं अन्य पक्ष के उत्पादों या परियोजनाओं से जुड़ा है जो कुबेरनेट्स द्वारा आवश्यक कार्यक्षमता प्रदान करते हैं। कुबेरनेट्स परियोजना के लेखक इन अन्य पक्ष के उत्पादों या परियोजनाओं के लिए जिम्मेदार नहीं हैं। अधिक जानकारी के लिए यह <a href="https://github.com/cncf/foundation/blob/master/website-guidelines.md" target="_blank">CNCF वेबसाइट दिशानिर्देश</a> पृष्ठ पढ़े।"</p><p>कोई नई अतिरिक्त अन्य पक्ष लिंक जोड़ने से पहले यह पृष्ठ <a href="/docs/contribute/style/content-guide/#third-party-content">विषय सूची मार्गदर्शन</a> पृष्ठ पढ़के ही परिवर्तन का प्रस्ताव प्रस्तुत करें।</p>"""
```

Comment:
```suggestion
other = """<p>इस पृष्ठ की वस्तुओं अन्य पक्ष के उत्पादों या परियोजनाओं से जुड़ा है जो कुबेरनेट्स द्वारा आवश्यक कार्यक्षमता प्रदान करते हैं। कुबेरनेट्स परियोजना के लेखक इन अन्य पक्ष के उत्पादों या परियोजनाओं के लिए जिम्मेदार नहीं हैं। अधिक जानकारी के लिए यह <a href="https://github.com/cncf/foundation/blob/master/website-guidelines.md" target="_blank">CNCF वेबसाइट दिशानिर्देश</a> पृष्ठ पढ़े।"</p><p>कोई नई अतिरिक्त अन्य पक्ष लिंक जोड़ने से पहले यह पृष्ठ <a href="/docs/contribute/style/content-guide/#third-party-content">विषय मार्गदर्शक</a> पृष्ठ पढ़के ही परिवर्तन का प्रस्ताव प्रस्तुत करें।</p>"""
```

Reply:
I have added the fix for `Content Guide` as suggested. PTAL. Thanks.